Package.swift

```diff
@@ -24,17 +24,21 @@ let package = Package(
                 .product(name: "Fluent{{fluent.db.module}}Driver", package: "fluent-{{fluent.db.url}}-driver"),{{/fluent}}{{#leaf}}
                 .product(name: "Leaf", package: "leaf"),{{/leaf}}
                 .product(name: "Vapor", package: "vapor"),
-            ]
+            ],
+            swiftSettings: swiftSettings
         ),
-        .testTarget(name: "AppTests", dependencies: [
-            .target(name: "App"),
-            .product(name: "XCTVapor", package: "vapor"),
-
-            // Workaround for https://github.com/apple/swift-package-manager/issues/6940
-            .product(name: "Vapor", package: "vapor"),{{#fluent}}
-            .product(name: "Fluent", package: "Fluent"),
-            .product(name: "Fluent{{fluent.db.module}}Driver", package: "fluent-{{fluent.db.url}}-driver"),{{/fluent}}{{#leaf}}
-            .product(name: "Leaf", package: "leaf"),{{/leaf}}
```

Comment:
I tested this on Ubuntu with `swift build --build-tests` and confirmed that the workaround was required with swift 5.9.0 and 5.9.1, but has been fixed and is no longer needed in 5.9.2 and 5.10.0.